storybookjs/storybook
# #2445 Add min, value, and max labels to range knob

Issue: #2445 

## What I did
Added a simple display for the range slider for knobs of type Number: minimum on the left of the slider, and current value and maximum on the right of the slider.

![storybook_range_labels](https://user-images.githubusercontent.com/10188148/36887406-f61443b4-1da5-11e8-89e5-00dc3442db00.jpg)


## How to test
Add a knob to a story: of type 'number' with range set to true. View the knobs panel for the story.
Expected: A slider appears with the minimum, current value, and maximum. The current value updates as you move the slider.

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
No (Appears in "official storybook" example).

Does this need an update to the documentation?
No
